go.mod

```diff
@@ -1,6 +1,8 @@
 module knative.dev/eventing-ceph
 
-go 1.18
+go 1.21
+
+toolchain go1.21.6
```

Comment:
This should not be here.
--> Brought up https://github.com/knative-extensions/eventing-ceph/pull/418

/hold